modules/oc-mirror-building-image-set-config-v2.adoc

```diff
@@ -2,31 +2,53 @@
 //
 // * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
 
-:_mod-docs-content-type: CONCEPT
+:_mod-docs-content-type: PROCEDURE
 [id="oc-mirror-building-image-set-config-v2_{context}"]
-= Building the image set configuration
+= Creating the image set configuration
 
-The oc-mirror plugin v2 uses the image set configuration as the input file to determine the required images for mirroring.
+Before you can use the oc-mirror plugin v2 to mirror image sets, you must create an image set configuration file.
```

Comment:
should we say images instead of image sets ?

Reply:
I cannot remember, but this might have been intentional word choice as a result of discussion with Sherine. @sherine-k do you know if one wording or another would be more accurate?

Reply:
okay, let us wait to hear from @sherine-k on that.

Reply:
Both are ok, I guess you can go with images. Thanks